jobs/config.json

```diff
@@ -3206,6 +3206,7 @@
       "--extract=ci/latest",
       "--gcp-master-image=gci",
       "--gcp-node-image=gci",
+      "--gcp-nodes=2000",
```

Comment:
2001? (with the heapster node)

Reply:
I changed that on purpose (for 5000-node we also have 5000, not 5001).

Reply:
Ok, SG